libraries/core/src/main/java/com/paritytrading/philadelphia/FIXConnection.java

```diff
@@ -548,6 +548,12 @@ public void message(FIXMessage message) throws IOException {
             }
 
             if (msgSeqNum != rxMsgSeqNum) {
+                // Sometimes a logout request can be paired with the 'MsgSeqNum too low' error
```

Comment:
In addition, could you please move this logic into `FIXConnection#handleTooLowMsgSeqNum()`? This is so that it we can keep this special situation off the fast path in `MessageHandler#message()`.